Added a new configuration entry to auto assign the @eng group

integral-llc opened the pull request:
### Motivation

 The motivation for the changes in this PR is to have the `@eng` group auto-assigned by the `dependabot`

< Ticket status, e.g. "fixes #issue number" > 
fixed ticket #1407

jcape reviewed (changes requested):
This should use [`reviewers`](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#reviewers), not `assignees`, and should be set for all of the "package-ecosystem" groups.

integral-llc commented:
> This should use [`reviewers`](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#reviewers), not `assignees`, and should be set for all of the "package-ecosystem" groups.

thx, wasn't sure. Updated the field and added across all groups.

garbageslam commented:
is it going to work with `eng` or does it need to be `mobilecoinfoundation/eng` ?

integral-llc commented:
> is it going to work with `eng` or does it need to be `mobilecoinfoundation/eng` ?

actually not sure about this. If I check the dropdown that GH is showing, I see the full name. @jcape ideas?

remoun commented:
Using the full name (incl. org) seems safe and not too verbose

jcape reviewed (approved):
LGTM, feel free to merge after CI passes successfully.
